k8s/helm/elasticsearch/minikube/Makefile

```diff
@@ -0,0 +1,17 @@
+default: test
+
+RELEASE := helm-es-minikube
+TIMEOUT := 1200s
+
+install:
+	helm upgrade --debug --wait --timeout=$(TIMEOUT) --install  --values ../values.yaml --values values.local.yaml $(RELEASE) ../../../../charts/elasticsearch/dependencies/helm-charts/elasticsearch/
+
+test: install
+	helm test $(RELEASE)
+
+purge:
+	helm del $(RELEASE)
+	kubectl delete pvc --all
```

Comment:
I'm quite uncomfortable having this `kubectl delete pvc --all` etc so close to production things and.
It just feels like one slip of auto complete could try and delete all the PVCs right now for example.
But maybe I'm worrying too much :P

Reply:
I totally agree, was just so we could run this to some extent and try it out. I don't mind dropping all of that stuff completely.

Reply:
I really like the idea of being able to run all of the parts in this repo locally too, so lets leave this here as a reminder for now?
Though it also might not work any more as I made changes without testing it on minikube locally.

I don't have minikube locally, but I do have a local k8s thing, but i don't use it much.
This conversation makes me remember https://kind.sigs.k8s.io/ again